packages/neutron-sdk/src/interchain_queries/mod.rs

```diff
@@ -12,8 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-pub mod error;
 pub mod helpers;
 pub mod queries;
-pub mod register_queries;
+mod register_queries;
 pub mod types;
+
+pub use register_queries::{
```

Comment:
why not simply pub mod register_queries? here in use you simply enumerated all pub fns from register_queries

Reply:
that's why! https://github.com/neutron-org/neutron-contracts/pull/19#discussion_r960450594